modules/postprocessing/src/processors/depthoffield.cpp

```diff
@@ -212,8 +212,15 @@ void DepthOfField::process() {
             std::swap(nextOutImg_, prevOutImg_);
         }
         evalCount_++;
-        dispatchFront([this, camera, maxEvalCount, focusDepth]() {
-            moveCamera(camera, maxEvalCount, focusDepth);
+        dispatchFront([weakSelf = weak_from_this(), maxEvalCount, focusDepth]() {
```

Comment:
this pattern should probably be mentioned in the docs

Reply:
Well I would rather that know one did anything like this ;)